web/css/review-form.css

```diff
@@ -1,5 +1,36 @@
+/* media queries */
+
+@media screen and (min-width: 1025px) {
+  .validator{
```

Comment:
I haven't tried, but I think you could leave the variables in the normal definitions, and only overwrite it in the `max-width: 1024px` media query. I think that would be cleaner. The same would go for the other definitions.